tests/generation/test_configuration_utils.py

```diff
@@ -99,7 +99,6 @@ class ConfigPushToHubTester(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         cls._token = TOKEN
-        set_access_token(TOKEN)
```

Comment:
This is removed. In the PR description, you mentioned `use set_git_credential if needed`. I am wondering why this PR doesn't use `set_git_credential`?

Reply:
I _think_ `set_access_token` was not needed in the first place. I suspect that it was used multiple times because of copy-pasting legacy code even though the tests do not need to authenticate with git. I have not reviewed in detail all the impacted tests but I thought that if the tests are passing, then it was not really needed (hence not using `set_git_credential`).

Maybe to be double-checked as I'm not used to `transformers`'s CI.

Reply:
Yes I think so. Now all the tests seems to use the HTTP endpoints which is nice :)